From f62ece4ebdbfa08c3813f2c3d65ced892865ab6b Mon Sep 17 00:00:00 2001
From: Tim Boden <mail@timboden.de>
Date: Sat, 5 Nov 2022 01:37:15 +0100
Subject: [PATCH] tiki

---
 premake5.lua      | 2 +-
 src/_manifest.lua | 2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/premake5.lua b/premake5.lua
index f9f2c8bb..254975ba 100644
--- a/premake5.lua
+++ b/premake5.lua
@@ -195,7 +195,7 @@
 			consumewinrtextension "false"
 
 	project "Premake5"
-		targetname  "premake5"
+		targetname  "premake_tb"
 		language    "C"
 		kind        "ConsoleApp"
 		includedirs { "contrib/lua/src", "contrib/luashim" }
diff --git a/src/_manifest.lua b/src/_manifest.lua
index 4b5acc62..76cd81be 100644
--- a/src/_manifest.lua
+++ b/src/_manifest.lua
@@ -68,4 +68,6 @@
 		"actions/clean/_clean.lua",
 
 		"_premake_init.lua",
+
+		"../tiki_build.lua"
 	}
-- 
2.21.0.windows.1

