diff --git forkSrcPrefix/src/_manifest.lua forkDstPrefix/src/_manifest.lua
index 79396b1ed72805918666f94c09fd9e761e9bff24..62352b3a443846ccac603d0eafc0d67108b0996e 100644
--- forkSrcPrefix/src/_manifest.lua
+++ forkDstPrefix/src/_manifest.lua
@@ -72,4 +72,6 @@
 		"actions/clean/_clean.lua",
 
 		"_premake_init.lua",
+
+		"../tiki_build.lua"
 	}
diff --git forkSrcPrefix/premake5.lua forkDstPrefix/premake5.lua
index 64a0f9fa59787b0d69c3c2509c9b3b6e4944332e..e1c0dd74a17ce042e38ce995eb20b3ac4a1a391b 100644
--- forkSrcPrefix/premake5.lua
+++ forkDstPrefix/premake5.lua
@@ -282,7 +282,7 @@
 			consumewinrtextension "false"
 
 	project "Premake5"
-		targetname  "premake5"
+		targetname  "premake_tb"
 		language    "C"
 		kind        "ConsoleApp"
 
